tests/src/test/scala/org/http4s/MediaRangeSpec.scala

```diff
@@ -41,6 +42,6 @@ class MediaRangeSpec extends Http4sSpec {
     }
   }
 
-  checkAll("Eq[MediaRange]", EqTests[MediaRange].eqv)
+  checkAll("Order[MediaRange]", OrderTests[MediaRange].order)
```

Comment:
Was necessary so we could get a Cogen for a set of them.  HTTP doesn't have anything to say about it, and maybe it should be demoted to a test package.